pages/Account.qml

```diff
@@ -136,6 +138,7 @@ Rectangle {
 
                 MoneroComponents.TextPlain {
                     id: unlockedBalanceAll
+                    Layout.rightMargin: 87
```

Comment:
Should be 80

Reply:
my alignment is correct with 87:
![image](https://user-images.githubusercontent.com/45968869/96346321-69723600-109b-11eb-91f9-3ab31d664c83.png)